api-documentation.js

```diff
@@ -452,6 +446,21 @@ class ApiDocumentation extends EventsTargetMixin(AmfHelperMixin(LitElement)) {
     this.requestUpdate('serversCount', old);
   }
 
+  get shouldRenderSelector() {
```

Comment:
This property is a public API for a component. I think you will admit that this is a weird name for a HTML attribute / property? More likely this would be `selectedRedered`. It's not `renderSelector` because this is not a setter. It just states the fact.

Reply:
Can also be `rendersSelector`.

Reply:
since it is a boolean, I usually try to name them as if they were a question that has `true` or `false` as a response. I like `rendersSelector`.